libs/state/README.md

```diff
@@ -83,59 +74,62 @@ nx migrate @rx-angular/state
 
 ## Usage
 
-[Usage Documentation](https://rx-angular.io/docs/state/setup)
+Local Provider (recommended): Use RxState as a local provider in your component to make use of Angular's Dependency Injection.
 
-## Testing
+```ts
+@Component({
+  ...
+  providers: [RxState],
+})
+export class RxStateInjectionComponent {
+  value$ = this.state.select();
 
-[Testing](https://rx-angular.io/docs/state/testing)
+  constructor(public state: RxState<{ foo: string }>) {}
+}
+```
 
-## API
+Inheritance: Use RxState by extending it in your component.
 
-[API Documentation](https://rx-angular.io/docs/state/api)
+```ts
+export class RxStateInheritanceClass extends RxState<{ foo: string }> {
+  value$ = this.select();
 
-## Tutorials
+  constructor() {
+    super();
+  }
+}
+```
```

Comment:
The API section maybe does make more sense if you move the `Usage` part on top of it. I think it should be the very first thing after the description.

Reply:
Agree